src/core/2d_primitives.js

```diff
@@ -47,30 +45,46 @@ require('./error_helpers');
  *
  * <div>
  * <code>
- * arc(50, 50, 80, 80, 0, PI+QUARTER_PI, OPEN);
+ * //Two corresponding arcs drawn in OPEN mode
+ * arc(50, 45, 70, 70, PI+QUARTER_PI, 0, OPEN);
+ * arc(45, 55, 70, 70, 0, PI+QUARTER_PI, OPEN);
  * </code>
  * </div>
  *
  * <div>
  * <code>
- * arc(50, 50, 80, 80, 0, PI+QUARTER_PI, CHORD);
+ * //Two corresponding arcs drawn in CHORD mode
+ * arc(50, 45, 70, 70, PI+QUARTER_PI, 0, CHORD);
+ * arc(45, 55, 70, 70, 0, PI+QUARTER_PI, CHORD);
  * </code>
  * </div>
  *
  * <div>
  * <code>
- * arc(50, 50, 80, 80, 0, PI+QUARTER_PI, PIE);
+ * //Two corresponding arcs drawn in PIE mode
+ * arc(50, 45, 70, 70, PI+QUARTER_PI, 0, PIE);
+ * arc(45, 55, 70, 70, 0, PI+QUARTER_PI, PIE);
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * //Arcs do not need to be drawn on a circle
+ * //Two corresponding arcs with different heights.
+ * arc(45, 50, 70, 70, HALF_PI, PI + HALF_PI, CHORD);
+ * arc(55, 50, 70, 35, PI + HALF_PI, HALF_PI, CHORD);
+ * </code>
+ * </div>
  * @alt
- *shattered outline of an ellipse with a quarter of a white circle bottom-right.
- *white ellipse with black outline with top right missing.
- *white ellipse with top right missing with black outline around shape.
- *white ellipse with top right quarter missing with black outline around the shape.
+ * shattered outline of an ellipse with a quarter of a white circle bottom-right.
```

Comment:
do these accessibility descriptions need to be updated to reflect the changes in the examples?